### DISCUSSION TITLE
Related to "Creating shareable "albums" from Niffler data sets"

### DISCUSSION THREAD

**Discussion by ayushkoli772** [Emory-HITI/Niffler#387] [Q&A]
Hello Mentors! I'm Ayush Koli, a sophomore currently pursuing Computer Science. My skill set matches with description of project (Python, Java, MongoDB) and also I'm acquainted with data analysis. I'm very excited to start working and contribute first time to OSS. Unfortunately I have never worked with "Niffler dataset", but I'm eager to learn new things and contribute along with you. Can you please provide me with details and skill set required to prove myself. Waiting for your reply. Thank you.

**Comment by pradeeban** [comment]
Niffler uses standard DICOM images it receives from PACS or scanners. Therefore, any DICOM images you can find from the Cancer Imaging Archive (TCIA) or similar sources should work. Going through the main README and the README of the modules should help you get started.

**Reply by ayushkoli772** [reply]
Sorry for late reply.
I will go through the README as you mentioned. I'm new to OSS contribution so there are many things that I'm clueless about but I'm eager to start working on this project. I'll surely get back to you in few days after getting idea about Niffler.
Thank you for replying.
